runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -166,4 +189,80 @@ public void onGameStateChanged(GameStateChanged event)
 			ladders.clear();
 		}
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.BARROWS_REWARD_GROUP_ID && config.showChestValue())
+		{
+			ItemContainer barrowsRewardContainer = client.getItemContainer(InventoryID.BARROWS_REWARD);
+			List<Integer> itemIds = new ArrayList<>();
```

Comment:
this list is just `itemMap.values()`. I think `getItemPriceBatch` should really take a `Collection`..